spec/react-masonry-component-test.js

```diff
@@ -101,8 +103,8 @@ describe('React Masonry Component', function() {
     });
 
     it('should allow custom props', function() {
-        const handler = () => {};
-        const component = TestUtils.renderIntoDocument(<MasonryComponent onClick={handler} test="testProp"/>);
+        const handler = function() {};
+        const component = TestUtils.renderIntoDocument(<MasonryComponent onClick={handler}/>);
```

Comment:
The whole point of this test was to see if this component allowed custom props (This was before the unknown prop warnings were put in)

It may be that this test makes no sense, and this pattern is an anti-pattern.

Probably worth discussing it in the issues